Source/OIDAuthState.h

```diff
@@ -117,7 +117,9 @@ typedef void (^OIDAuthStateAuthorizationCallback)(OIDAuthState *_Nullable authSt
 @property(nonatomic, weak, nullable) id<OIDAuthStateErrorDelegate> errorDelegate;
 
 /*! @brief Convenience method to create a @c OIDAuthState by presenting an authorization request
-        and performing the authorization code exchange in the case of code flow requests.
+        and performing the authorization code exchange in the case of code flow requests. For
+        the hybrid flow, the caller should validate the id_token and c_hash, then call
+        [OIDAuthorizationService performTokenRequest] then [OIDAuthState updateWithTokenResponse]
```

Comment:
… c_hash, then perform the token request (@c OIDAuthorizationService.performTokenRequest:callback:)
and updating the OIDAuthState with the results (@c OIDAuthState.updateWithTokenResponse:error:).